requirements/edx/base.txt

```diff
@@ -322,19 +320,19 @@ django-mptt==0.14.0
     #   openedx-django-wiki
 django-multi-email-field==0.6.2
     # via edx-enterprise
-django-mysql==4.7.1
+django-mysql==4.8.0
     # via -r requirements/edx/base.in
 django-oauth-toolkit==1.3.2
     # via
     #   -c requirements/edx/../constraints.txt
     #   -r requirements/edx/base.in
-django-object-actions==4.0.0
+django-object-actions==4.1.0
     # via edx-enterprise
 django-pipeline==2.0.8
     # via -r requirements/edx/base.in
 django-pyfs==3.2.0
     # via -r requirements/edx/base.in
-django-ratelimit==3.0.1
+django-ratelimit==4.0.0
```

Comment:
https://django-ratelimit.readthedocs.io/en/stable/upgrading.html#from-3-x-to-4-0
Upgrade notes can be found on this link. Upgraded the code keeping these notes in view.